docs/RELEASE.md

```diff
@@ -1,12 +1,16 @@
 # Release process
 
+* Ensure you have a backup of all working files and then remove files not tracked by git
+  `git clean -xdf`. **NOTE**: this will delete all files in the tuf tree that aren't
+  tracked by git
 * Ensure `docs/CHANGELOG.md` contains a one-line summary of each [notable
   change](https://keepachangelog.com/) since the prior release
 * Update `setup.py` and `tuf/__init__.py` to the new version number vA.B.C
 * Test packaging, uploading to Test PyPI and installing from a virtual environment
+  (ensure commands invoking `python` below are using Python 3)
```

Comment:
I'm not sure we need this note (here and below) if the commands also say `python3`

Reply:
I think it's okay to be very clear about it. But I'm fine either way.

Reply:
I opted to be very clear, just in case readers are not diligently reading each step